elasticpypi/config.py

```diff
@@ -1,7 +1,26 @@
-import json
 import os
 
+WATCHED=[
+    'SERVICE',
+    'STAGE',
+    'BUCKET',
+    'TABLE',
+    'USERNAME',
+    'PASSWORD',
+    'OVERWRITE'
+]
+
+TRUTHY = ['TRUE', 'true', 1]
+FALSEY = ['FALSE', 'false', 0]
+
+# load config
+config = {k.lower(): v for k, v in os.environ.iteritems() if k in WATCHED}
+
+# check config
+for k, v in config.iteritems():
+    # assume env vars are always stringy
+    if v in TRUTHY:
+        config[k] = True
```

Comment:
I agree that it should be the user's responsibility. The problem I ran into was that env vars aren't typed when read in. Since I didn't want to muck with things outside config.py, I "recast" them as boolean. Is there a better way around that?